apps/web/lib/slots.ts

```diff
@@ -41,7 +41,8 @@ const getSlots = ({ inviteeDate, frequency, minimumBookingNotice, workingHours,
   const startOfDay = dayjs.utc().startOf("day");
   const startOfInviteeDay = inviteeDate.startOf("day");
   // checks if the start date is in the past
-  if (inviteeDate.isBefore(startDate, "day")) {
+
+  if (inviteeDate.isBefore(startDate, "hour")) {
```

Comment:
This was displaying a day as available as sometimes difference between two dates is < 24 hrs. But when doing timezones a available day for a owner can be 2 days available in other users tz.